.build/deploy-snapshot.sh

```diff
@@ -16,6 +16,6 @@ elif [ "$TRAVIS_BRANCH" != "$BRANCH" ]; then
   echo "Skipping snapshot deployment: wrong branch. Expected '$BRANCH' but was '$TRAVIS_BRANCH'."
 else
   echo "Deploying snapshot..."
-  mvn clean source:jar javadoc:jar deploy --settings=".build/settings.xml" -Dmaven.test.skip=true -Psonatype
+  mvn clean source:jar javadoc:jar deploy --settings=".build/settings.xml" -DwithJavadoc -Dmaven.test.skip=true -DwithStaging
```

Comment:
Why is it name withStaging while it's not a staging repo here.

Reply:
I just used the name that Sonatype is using (https://oss.sonatype.org/service/local/staging/deploy/maven2/). 
Indeed when you publish to sonatype repo you are staging your artifacts there. There you can set your repo to automatically publish artifacts into maven.central, but you can also set it to not do this.